src/libraries/System.Private.CoreLib/src/System/Text/Rune.cs

```diff
@@ -911,6 +916,33 @@ public override string ToString()
 #endif
         }
 
+#if SYSTEM_PRIVATE_CORELIB
+        bool ISpanFormattable.TryFormat(Span<char> destination, out int charsWritten, ReadOnlySpan<char> format, IFormatProvider? provider)
+        {
```

Comment:
Can this entire method be a wrapper around `return TryEncodeToUtf16(destination, out charsWritten)`? The code looks identical.

Reply:
Even better :)